MediaBrowser.Model/Entities/ImageType.cs

```diff
@@ -46,7 +46,7 @@ public enum ImageType
         Box = 7,
 
         /// <summary>
-        /// The screenshot.
+        /// [Deprecated] The screenshot.
         /// </summary>
         Screenshot = 8,
```

Comment:
Use the `[Obsolete]` attribute with a description

Reply:
Coming most recently from a language/project where deprecations were handled by convention in the docs, I missed that there is an actual attribute for it.